test/module/irohad/common/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+add_executable(config_loader_test
```

Comment:
Andrew's  commentary is very important. Pls, rework cmake file with this feature. Example of usage see [here](https://github.com/hyperledger/iroha/blob/develop/test/module/irohad/torii/processor/CMakeLists.txt)